server/app/models/OrderModel.ts

```diff
@@ -0,0 +1,23 @@
+import { Document, Schema } from 'mongoose';
+import { MaterialTypes, materialTypes } from '../entities/Material';
+
+export interface IOrder extends Document {
+  material: materialTypes;
+  quantity: number;
+  cost: number;
+  deliveryDate: Date;
+  orderDate: Date;
+  manufactureName: string;
+  status: string;
+  note: string;
+}
+export const OrderSchema: Schema = new Schema({
+  material: { type: Schema.Types.String, enum: MaterialTypes, required: true },
+  quantity: { type: Schema.Types.Number, required: true, min: 1 },
+  cost: { type: Schema.Types.Number, required: true, min: 1 },
```

Comment:
I thought we were going to hard code the cost of each material with the material types right?

Reply:
I saw that Mongoose has a Map type and thought about having one field for cost and material. It was kind of complicated to implement so I just made cost to not be required and whenever you create/update then it'll update the cost value from the predefined map

Reply:
Sounds good, yo sorry to bust your balls again but i just noticed you're missing 'leather' in material types LOL